s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3Client.scala

```diff
@@ -56,9 +56,14 @@ final class S3Client(s3Settings: S3Settings, system: ActorSystem, mat: Materiali
     impl.download(S3Location(bucket, key), Some(scalaRange)).asJava
   }
 
-  // #list-bucket
```

Comment:
docs/src/main/paradox/s3.md includes this snippet in the documentation. Perhaps those other methods should also be added to the documentation? What do you think?